Userland/Libraries/LibELF/Arch/aarch64/plt_trampoline.S

```diff
@@ -8,6 +9,56 @@
 .globl _plt_trampoline
 .hidden _plt_trampoline
 .type _plt_trampoline,@function
-_plt_trampoline: # (object, relocation_index)
-    # FIXME: Possibly incomplete.
-    ret
+
```

Comment:
Oh, and maybe put a link to https://maskray.me/blog/2021-09-19-all-about-procedure-linkage-table#aarch64 (or whatever you used) in a comment here :)

Reply:
The arm64 ABI is so well documented that I could work directly off of it. <https://github.com/ARM-software/abi-aa/blob/844a79fd4c77252a11342709e3b27b2c9f590cf1/sysvabi64/sysvabi64.rst#93procedure-linkage-table>